tests/integ/test_dataframe.py

```diff
@@ -269,6 +271,13 @@ def test_distinct(session):
     res = df.select(col("v")).distinct().sort(["v"]).collect()
     assert res == [Row(None), Row(1), Row(2), Row(3), Row(4), Row(5)]
 
+    if not local_testing_mode:
```

Comment:
just a questions: what is the purpose of this part of test? Isn't the value check above sufficient for the test?

Reply:
This test is checking if the query generated by df.distinct() uses `GROUP BY` which is the old implementation when  the optimization is turned-off and it uses `SELECT DISTINCT` when the optimization is turned on.